asv_runner/util.py

```diff
@@ -107,6 +107,8 @@ def human_float(value, significant=3, truncate_small=None, significant_zeros=Fal
     ("h", 60 * 60),
     ("d", 60 * 60 * 24),
     ("w", 60 * 60 * 24 * 7),
+    ("M", 60 * 60 * 24 * 7 * 52 / 12),
+    ("q", 60 * 60 * 24 * 7 * 52 / 4),
```

Comment:
Does this need more handling? e.g. does it need to change the output format?

Reply:
I don't think so; it should ll be in seconds